addons/metadata.themoviedb.org.python/python/lib/tmdbscraper/tmdb.py

```diff
@@ -138,9 +138,13 @@ def _parse_media_id(title):
     return None
 
 def _get_movie(mid, language=None, search=False):
-    details = None if search else \
-        'trailers,images,releases,casts,keywords' if language is not None else \
-        'trailers'
+    if search:
+        details = None
```

Comment:
I am not sure if this add-on doesn't have an upstream location. @razzeee @pkscout do you guys know?
It feels like any change to these addons should go upstream first and then we just sync.

Reply:
https://github.com/xbmc/metadata.themoviedb.org.python